guides/source/configuring.md

```diff
@@ -140,6 +140,8 @@ defaults to `:debug` for all environments. The available log levels are: `:debug
 
 * `secrets.secret_key_base` is used for specifying a key which allows sessions for the application to be verified against a known secure key to prevent tampering. Applications get `secrets.secret_key_base` initialized to a random key present in `config/secrets.yml`.
 
+* `config.secrets_cipher` allows you to change the cipher used to protect the `config/secrets.yml` file. This defaults to `"aes-128-gcm"`. Versions of OpenSSL earlier than 1.0.01, unfortunately, don't support GCM mode, so degrading this to `"aes-128-cbc"` may be necessary. If you find yourself in this situation, strongly consider updating OpenSSL to a modern version if at all possible rather than weakening this setting.
```

Comment:
I'd totally expect `config/secrets.yml` to be the decrypted `config/secrets.enc`, too… even knowing what's actually happening and why. 😊

The mixed heritage behind this feature is muddling things up. We have a mix of approaches like

- `config/secrets.yml` is a gitignored file that's copied in as needed in dev and on production deploy
- `config/secrets.yml` is a checked-in file that doesn't contain actual secrets, just ERB preprocessing that references environment variables with actual secrets
- `config/secrets.yml` is a checked-in file that contains actual secrets, but only for dev/test.

And they're competing for mindshare and, literally, file naming.

In any case, this is way out of scope for _this_ PR. Good topic for further investigation.